### ISSUE TITLE
Uncaught (in promise) DOMException: The operation is insecure.

### ISSUE THREAD

**Issue by nickelswitte** [blaise-io/xssnake#8]
When starting a game, it gets stuck in "Connecting..."

Inspecting the browser console, there is the following error: 
`Uncaught (in promise) DOMException: The operation is insecure.`

The log from the application side is free of any errors.

How to fix this?

![grafik](https://user-images.githubusercontent.com/36443026/222973763-d1728aaf-d906-46b2-8a78-3239247faf6f.png)

**Comment by nickelswitte** [comment]
@blaise-io Is there an update on this?

**Comment by blaise-io** [comment]
@nickelswitte no, sorry. I'm not working on this project for the foreseeable future.
